lms/static/js/instructor_dashboard/student_admin.js

```diff
@@ -474,7 +476,10 @@
                 success: this.clear_errors_then(function() {
                     return alert(fullSuccessMessage);  // eslint-disable-line no-alert
                 }),
-                error: statusAjaxError(function() {
+                error: statusAjaxError(function(response) {
+                    if (response.responseText === 'Specified module does not support rescoring.') {
```

Comment:
@nasthagiri For my own edification: Are you saying that this specific error should not return an error at all to AJAX, and instead have "fake success", with a variety of messages for cases, like this error?